tests/dir.props

```diff
@@ -77,6 +78,13 @@
     <RestorePackages>false</RestorePackages>
   </PropertyGroup>
 
+  <!-- If we want to overwrite the desired CoreCLR package version, we need to get the new version from the generated props file in bin/obj -->
+  <Import Condition="'$(OverwriteCoreClrPackageVersion)' == 'true'" Project="$(BaseIntermediateOutputPath)BuildVersion-$(OfficialBuildId).props" />
+
+  <PropertyGroup Condition="'$(OverwriteCoreClrPackageVersion)' == 'true'">
+    <PackageVersionArg>/p:CoreClrPackageVersion=2.1.0-preview1-$(BuildNumberMajor)-$(BuildNumberMinor)</PackageVersionArg>
```

Comment:
'PackageVersion' isn't available during this part of the build, since root dir.props isn't imported by anything in the tests directory. Is there a reason tests/dir.props doesn't import root dir.props?